.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,20 +1,3 @@
-<!--
```

Comment:
Why was this deleted?

Reply:
> Why was this deleted?

Clarified with @DarthHater: "the header isn’t really needed on the file you asked about, that’s a bug template and it’s kinda unnecessary "

Reply:
Also, the header shows up in the actual bug report - e.g. when you create a new bug report, BAM - the header text is shown. yuck.